0200-background.Rmd

```diff
@@ -1,377 +1,375 @@
-
-```{r}
-##`r pscis_historic_phase1 %>% filter(watershed_group_code %ilike% 'MORR' & assessment_date < '2020-01-01') %>% nrow()`
-```
+# Background
+
+The study area includes the Bulkley River, Zymoetz River, Kispiox River and Morice River watershed groups (Figure \@ref(fig:overview-map)) and is within the traditional territories of the Gitxsan and Wet'suwet'en.
+
+<br>
+
+## Wet'suwet'en
+
+Wet'suwet'en hereditary territory covers an area of 22,000km^2^ including the Bulkley River and Morice River watersheds and portions of the Nechako River watershed. The Wet'suwet'en people are a matrilineal society organized into the Gilseyhu (Big Frog), Laksilyu (Small Frog), Tsayu (Beaver clan), Gitdumden (Wolf/Bear) and Laksamshu (Fireweed) clans. Within each of the clans there are are a number of kin-based groups known as Yikhs or House groups. The Yikh is a partnership between the people and the territory. Thirteen Yikhs with Hereditary Chiefs manage a total of 38 distinct territories upon which they have jurisdiction. Within a clan, the head Chief is entrusted with the stewardship of the House territory to ensure the Land is managed in a sustainable manner. Inuk Nu'at'en (Wet'suwet'en law) governing the harvesting of fish within their lands are based on values founded on thousands of years of social, subsistence and environmental dynamics. The Yintahk (Land) is the centre of life as well as culture and it's management is intended to provide security for sustaining salmon, wildlife, and natural foods to ensure the health and well-being of the Wet'suwet'en [@officeofthewetsuweten2013Wetsuwet; @OfficeoftheWetsuweten2021; @flnrord2017NaturalResource].
+
+<br>
+
+## Gitxsan
+
+Gitxsan means "People of the River Mist". The Gitxsan Laxyip (traditional territories) covers an area of 33,000km^2^ within the Skeena River and Nass River watersheds. The Laxyip is governed by 60 Simgiigyet (Hereditary Chiefs), within the traditional hereditary system made up of Wilps (House groups). Anaat are fisheries tenures found throughout the Laxyip. Traditional governance within a matrilineal society operates under the principles of Ayookw (Gitxsan law) [@GitxsanHuwilpGovernment2021].  Its people reside in the communities of Hazelton, Kispiox and Glen Vowell (the Eastern Gitxsan bands) and Kitwanga, Kitwankool and Kitsegukla (the Western Gitxsan). The original Gitxsan economy depended heavily on the trade of fish and other natural resources with neighbouring First Nations along "grease trails," which are routes named so because they carried the refined oil of the oolichan, a fish that resembles a smelt that is common in some areas of British Columbia. They are also known for their range of traditional arts [@j_v_powelletal_GitxsanCanadian].
```

Comment:
I thought you wanted to take out the line that said "its people"? You could just replace it with "the Gitxsan". And to avoid being repetitive you could change the following sentence to start with "Their economy depended..."

Reply:
i forgot about that one. I think you need to press a different button to confirm it is "reviewed" so that I can merge.  not sure though but have a look.  I can bypass it but will be good for you to know how to do it

Reply:
Ok I approved the changes